modules/basegl/src/processors/volumemasker.cpp

```diff
@@ -33,25 +33,56 @@
 namespace inviwo {
 
 // The Class Identifier has to be globally unique. Use a reverse DNS naming scheme
-const ProcessorInfo VolumeMasker::processorInfo_{
-    "org.inviwo.VolumeMasker",  // Class identifier
-    "Volume Masker",            // Display name
-    "Volume Operation",         // Category
-    CodeState::Experimental,    // Code state
-    Tags::GL,                   // Tags
-    R"(Applies the mask, given in the second volume, to the input volume.)"_unindentHelp};
+const ProcessorInfo VolumeMasker::processorInfo_{"org.inviwo.VolumeMasker",  // Class identifier
+                                                 "Volume Masker",            // Display name
+                                                 "Volume Operation",         // Category
+                                                 CodeState::Experimental,    // Code state
+                                                 Tags::GL,                   // Tags
+                                                 R"(
+Mask `inputVolume` with the second `mask` volume through multiplication. The output volume inherits all 
+parameters from the input volume including value/data ranges and transformations. Its contents are set to
+```glsl
+output = input * mask.r,
+```
+where the mask volume is assumed to be a single-channel volume.
+
+**Note:** 
+Depending on the data ranges of the input volume and the mask volume, the result may lie outside the 
```

Comment:
Any reason for not adjusting the range instead?

Reply:
Think the prime reason is that you would not expect the the masking to change it...

Reply:
That is why multiplication might not be a good idea :)

Reply:
And without (binary) multiplication masking is no longer applicable. Meaning you should then probably not use this processor.

Reply:
I don’t understand. This is what I had in mind:
`val = val ? mask : 0`

Reply:
You probably meant `val = mask ? val : 0`. But this is the same as multiplying by 0/1.

Reply:
If the mask is binary (0 and 1) that would be the same. Otherwise the mask (with values in a range) could benefit from the multiplication. The question could be what kind of mask is non-binary in this case. I am not familiar much in this domain but probably there might be a mask with non-binary values. any suggestion ?

Reply:
An other question is what happens with the interpolation in when using the ternary operator... multiplication would handle that much better...

Reply:
Agree, multiplication is much better if you want smooth masking.
One could put a requirement on the mask volume value range to be in [0 1] to ensure that the processor does what is intended.